app/sim_test.go

```diff
@@ -304,6 +306,17 @@ func testAndRunTxs(app *App, config simulation.Config) []simulation.WeightedOper
 			}(nil),
 			pricefeedsimops.SimulateMsgUpdatePrices(app.pricefeedKeeper),
 		},
+		{
+			func(_ *rand.Rand) int {
+				var v int
+				ap.GetOrGenerate(app.cdc, OpWeightMsgCdp, &v, nil,
+					func(_ *rand.Rand) {
+						v = 10000 // TODO
```

Comment:
this is the percentage of simulation (i.e  from all the times this operation is selected, this is the % of the times this message is actually run)

Reply:
```suggestion
						v = 100
```